framework/queries/postgres_db.py

```diff
@@ -99,3 +100,51 @@ def get_random_users(self, quantity: int = 1) -> List[dict]:
             """
         )
         return response
+
+    def create_user(self, user: dict) -> None:
+        """Inserting user into database
+
+        Args:
+            user: user data
+        """
```

Comment:
can you add a description to the data body?
```
"""Inserting user into database
    Args:
        user: user data:
                    - id - user of id;
                    - first_name - first name of user;
                    - ...
                    - hashed_password - hash of password for user.
"""
```

Reply:
Fixed.